Rethinking_2/Chp_07.ipynb

```diff
@@ -33,7 +33,7 @@
    "source": [
```

Comment:
<p>Same comment here about <code>FutureWarning: Using from_pymc3 without the model will be deprecated in a future release</code> : just add <code>return_inferencedata=True</code> in the <code>sample</code> calls</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/110/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='79'/>

Reply:
<p>ok</p>